Manifest.toml

```diff
@@ -1,1616 +0,0 @@
-# This file is machine-generated - editing it directly is not advised
```

Comment:
@Xiaoyan-Li I forgot to mention that we probably don't need this committed to the repo since StockFlow.jl is a library rather than an application. See the discussion @ https://discourse.julialang.org/t/does-manifest-toml-belong-in-the-repository/12029 and let me know if you'd prefer to restore it.

It seems to me like it'd be a common source of merge conflicts.